### PR TITLE
Completly removing MapFish print

### PR DESCRIPTION
**Pull request by procrastinatio**
MapFish print v2 will continue its life in https://github.com/geoadmin/service-print

This PR removes all its artefacts in this project.

So merge, and deploy https://github.com/geoadmin/service-print/pull/4. Dito for https://github.com/geoadmin/mf-geoadmin3/pull/3365. 
Wenn triple tested and checked, merge that PR.

This replace PR https://github.com/geoadmin/mf-chsdi3/pull/2116 (deploy no print)

### CODE REVIEW

**Comment by loicgasser** [comment]
Don t forget the hooks and scripts

**Comment by loicgasser** [comment]
I would also remove the legacy targets in the configs files.

**Comment by loicgasser** [comment]
You can remove all that is under https://github.com/geoadmin/mf-chsdi3/tree/master/print

Adapt gitignore.

**Comment by gjn** [comment]
Legacy targets still needed for height/profile service.
Am 06.07.2016 4:15 nachm. schrieb Gasser Loïc <notifications@github.com>:I would also remove the legacy targets in the configs files.

**Comment by loicgasser** [comment]
huuu, what's the point of removing the print if we still need to deploy chsdi on all 4 instances?

We need a service-height then, as we discussed it with @procrastinatio if I am not mistaken.
Let's focus on that and Jessie migration. We can do both at the same time.

**Comment by procrastinatio** [comment]
@loicgasser _Service isolation_ (it's only the first step), it is stupid to deploy things that are not used, have not changed.

**service-alti** in progress https://github.com/geoadmin/service-alti

**Comment by gjn** [comment]
yes, that's the goal.

Anyway, removing print from mf-chsdi3 could already be done.

**Comment by loicgasser** [comment]
I will rebase/finish it and keep targets

**Comment by loicgasser** [comment]
ping @procrastinatio and @gjn for a review

File: Makefile [hunk 1]
```diff
@@ -440,10 +394,6 @@ fixrights:
 clean:
 	rm -rf production.ini
 	rm -rf development.ini
-	rm -rf apache/wsgi.conf
```
Comment (gjn) [inline Makefile]: I think we need to keep this in the clean.

File: Makefile [hunk 2]
```diff
@@ -440,10 +394,6 @@ fixrights:
...
-	rm -rf apache/tomcat-print.conf
-	rm -rf print/WEB-INF/web.xml
-	rm -rf apache/application.wsgi
```
Comment (gjn) [inline Makefile]: Same here, we need to keep this.

File: Makefile [hunk 3]
```diff
@@ -303,22 +273,6 @@ deploy/conf/00-branch.conf: deploy/conf/00-branch.conf.in
 	@echo "${GREEN}Creating deploy/conf/00-branch.conf...${RESET}"
 	${MAKO_CMD} --var "git_branch=$(GIT_BRANCH)" $< > $@
 
-apache/tomcat-print.conf.in:
```
Comment (gjn) [inline Makefile]: This file is not removed, from what I can see. Please remove it.

**Comment by gjn** [comment]
Added a couple of comments that I think need corrections. Also, a squash would be nice.

**Comment by loicgasser** [comment]
rebased, squashed and adapted to comments
Thks for the review

**Comment by loicgasser** [comment]
@gjin if we ever want to remove the useless target instances

**Comment by loicgasser** [comment]
@gjn

**Pull request merged**